tests/Core/Components.Tests.csproj

```diff
@@ -72,4 +72,31 @@
       <DependentUpon>FluentLocalizer.resx</DependentUpon>
     </Compile>
   </ItemGroup>
+
```

Comment:
Can you remove all of these lines? Already include using a global pattern.

Reply:
ok, lines removed from .csproj